larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -179,8 +179,8 @@ public int runScenarios(IbisContext ibisContext, String paramLogLevel, String pa
 		if (paramAutoScroll == null && paramLogLevel != null) {
 			config.setAutoScroll(false);
 		}
-		if (paramMultithreaded == null && paramLogLevel != null) {
-			config.setMultiThreaded(false);
+		if (("on".equals(paramMultithreaded) || Boolean.parseBoolean(paramMultithreaded)) && paramLogLevel != null) {
+			config.setMultiThreaded(true);
```

Comment:
I think this check should not depend on `paramLogLevel`. I also think we can revert the `"on"` check to a `null` check, and set the checkbox directly from `paramMultiThreaded != null`, 

Something like this:
```suggestion
		config.setMultiThreaded(paramMultithreaded != null);
```

Reply:
Due to the poor design of this method, you need to be able to change a value without larva running all scenario's.
Not why why this was like this, it's not been changed, and I do not intend to do so.